01/6._global_variable/main.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+int g_x = 100;
+
+void increase_number(void)
+{
+    int y = 200;
+    y += 100;
+    g_x += 100;
+}
+
+int main(void)
+{
+    printf("Value Of Global Variable: %d\n", g_x);
+    increase_number();
+    printf("Value Of Global Variable: %d\n", g_x);
+    /*printf("Value Of Local Variable: %d\n", y);*/

```

Comment:
is this comment left here on purpose? ignore me if it is on purpose

Reply:
It is